jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/AdministrationControllerTest.java

```diff
@@ -318,22 +319,48 @@ public void groupAdministrationPageShouldContainListOfGroups() throws Exception
                 .andExpect(model().attribute("groups", expected));
     }
 
+    private List<UserDto> buildTestUserDtoList() {
+        List<UserDto> result = new ArrayList<>();
+        result.add(new UserDto("user1", "user1@mail.ru", 1L));
```

Comment:
Use qala-datagen to fill the random values. Also you can create a method `randomUserDto()` and just invoke it multiple times from here.